Migration script/openIMIS migration v1.4.1 - v1.4.2.sql

```diff
@@ -1514,3 +1514,11 @@ AND LocationType = N'D'
 GO
 
 CREATE UNIQUE CLUSTERED INDEX CI_tblDistricts ON tblDistricts(DistrictId)   
+
+
+-- OP-153: Additional items in the Reports section
+
+IF COL_LENGTH('tblReporting', 'ReportMode') IS NULL
+BEGIN
+	ALTER TABLE tblReporting ADD ReportMode int NULL
```

Comment:
Should this have a default value? Valid also for the other scripts.

Reply:
From what I saw in the web app code, there is possibility that value 1 is for actual paid contributions and value 0 is for prescribed contributions, let me know if any of these modes should be assigned by default.

Reply:
You can put this as default 0.

Reply:
I've added default values to migration and to empty db creation scripts.